tests/Resources/OCP.resource

```diff
@@ -38,3 +38,13 @@ Verify Namespace Status
     @{NS_obj} =  OpenShiftCLI.Get  kind=Namespace  label_selector=${label}
     &{NS_obj_dictionary} =  Set Variable  ${NS_obj}[0]
     Should Be Equal  ${NS_obj_dictionary.status.phase}  Active
+
+Verify CR Status
+    [Documentation]  Verifies that a CR exists, that it's available
+    ...    and that the name matches
+    [Arguments]  ${CRD}  ${CR_name}
+    @{CR_obj} =  OpenShiftCLI.Get  kind=${CRD}
+    &{CR_obj_dictionary} =  Set Variable  ${CR_obj}[0]
```

Comment:
As per our style guide we should not have name with Capitals letter for limited scope variable. Could you please change it

Reply:
https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit#heading=h.4a2fb94fltk2

Reply:
I've added a draft, look it over when you have some time @bdattoma @tarukumar @jgarciao

Reply:
I've added my toughs to the document

Reply:
Agreement: we'll use lowercase for local keywords and parameters